src/main/java/com/google/sps/GetConfigProperties.java

```diff
@@ -32,11 +32,14 @@ public static String getPropertyValue(String filepath, String propertyName) thro
 
   /**
    * This function retrieves the value of the API Key from the app.properties file. 
-   * @return String                 This return the String value of the API key.
-   * @throws FileNotFoundException  This exception is thrown if the filepath given does not exist. 
-   * @throws IOException            This exception is thrown if the filepath and/or the propertyName do not exist. 
+   * @return String     This return the String value of the API key.
    */
-  public static String getApiKey() throws FileNotFoundException, IOException {
-    return getPropertyValue("app.properties", "api_key");
+  public static String getApiKey() {
+    try {
+      return getPropertyValue("app.properties", "api_key");
+    } catch (IOException e) {
+      e.printStackTrace();
```

Comment:
Returning a legal value that means 'error' (an empty string is a valid string, other examples are returning '-1' or even returning 'null') is an anti-pattern. It hides errors, often causing follow-on errors in other parts of the code that rely on the function in question, except then you won't have the context to understand what went wrong (printing a stacktrace is a poor substitute, it's easily missed). 

Either rethrow the exception (with a more useful error) or return an empty 'optional'.